Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java

```diff
@@ -190,7 +190,7 @@ protected void testGetActiveParticipantSource(LogEventRequestType request, Boole
         }
 
         assertEquals("NetworkAccessPointID mismatch", ipOrHost, sourceActiveParticipant.getNetworkAccessPointID());
-        assertEquals("SourceActiveParticipant requestor flag mismatch", isRequesting,
+        assertEquals("SourceActiveParticipant requestor flag mismatch", Boolean.TRUE,
```

Comment:
I prefer variables over hard-coded values, so I have to ask -- is there a problem with using "isRequesting"?